cypress-io/cypress
# fix: Styling of specs list at small breakpoints

- Closes #22626 

### User facing changelog
* Fix styling issues of Specs Explorer at narrow viewport widths

### Additional details
* Fix issue where spec rows aren't vertically centered at narrow viewport widths
  - This was caused by using CSS `visibility` rather than `display` which caused layout space to be preserved while hiding the item visually when we actually wanted the item to be fully removed from the layout flow at the lower breakpoint. It was being correctly handled in the header but not in the row items
* Per product request, increase min width of "New Spec" button to prevent wrapping. We may want to look into responsively scaling the `Button` component's padding & icon size but that feels like a larger issue to be addressed with UX

### Steps to test
1. Open a project with specs in the Cypress app, navigate to Specs Explorer
2. Narrow viewport to < 768px, observe spec rows remain vertically centered and button text does not wrap

### How has the user experience changed?
**Before**
![Screen Shot 2022-06-30 at 3 46 18 PM](https://user-images.githubusercontent.com/11482842/176775505-e64ece0a-bf31-485a-a4cf-70a75e909138.png)

**After**
![Screen Shot 2022-06-30 at 3 45 47 PM](https://user-images.githubusercontent.com/11482842/176775523-cde50bcd-165a-4b01-bfe3-f042f1d3fcf6.png)

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
